templates/Bookmarks.html

```diff
@@ -1,9 +1,83 @@
 {% extends 'base.html' %}
 
 {% block head %}
+<style>
+  main {
+    background-image: url(/static/images/signup-background2.png);
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+  }
 
+  body {
+    overflow-y: auto;
```

Comment:
this is likely the cost of the strange margin bug - i will fix in the design changes branch